lib/ui/playground/playground_present_button.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:farmsmart_flutter/ui/common/Alert.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class PlaygroundPresentButton extends StatelessWidget {
+  final Widget child;
+  final Function(Widget, BuildContext) present;
+
+  const PlaygroundPresentButton({
+    Key key,
+    Function(Widget, BuildContext) listener,
+    Widget child,
+  })  : this.present = listener,
+        this.child = child,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: RaisedButton(
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+        child: Text("Press Me"),
+        onPressed: () => present(child, context),
+      ),
+    );
+  }
+
+  static presentModal(Widget modal, BuildContext context) {
```

Comment:
@DavidFD005 this reminds me.. we need a Present model helper function that will take a child widget wrap in a navigation and present modally on top. whenever we present model flow we can then. use that.

Reply:
DONE https://github.com/farmsmart/farmsmart-flutter/pull/58/files#diff-e7ac1d6b885972a17dd9f514c4b70d25

Reply:
👍